### PR TITLE
🔎 Refactor plot_ragged function to improve clarity and documentation

### PR DESCRIPTION
**Pull request by selipot**
This PR is to clarify the docstring of ``plot_ragged``. @philippemiron let me know what you think.

### CODE REVIEW

File: clouddrift/plotting.py [hunk 1]
```diff
@@ -41,12 +41,12 @@ def plot_ragged(
     *args : tuple
         Additional arguments to pass to ``ax.plot``.
     colors : array-like
-        Colors to use for plotting. If colors is the same shape as longitude and latitude,
-        the trajectories are splitted into segments and each segment is colored according
-        to the corresponding color value. If colors is the same shape as rowsize, the
-        trajectories are uniformly colored according to the corresponding color value.
+        Values to map on the current colormap. If ``colors`` is the shape of ``rowsize``,
+        the data points of each row are uniformly colored according to the color value for the row.
```
Comment (philippemiron) [inline clouddrift/plotting.py:45]: Should we removed "uniformly" here? It is colored as a function of the values that you supply, not necessarily uniform.

**Review by philippemiron — APPROVED** [review]

File: clouddrift/plotting.py (hunk 1 above)
Comment (kevinsantana11) [reply clouddrift/plotting.py:45]: Maybe we could say:
```
Colors to use for plotting. If ``colors`` has the same shape as ``rowsize``,
each ``ob`` in a ragged array is colored according to the ``row`` it is associated with.
```

**Comment by philippemiron** [comment]
There was a comment that wasn't resolved...

**Comment by selipot** [comment]
Resolved now! I took a decision to move forward.

**Pull request merged**